From 434890171d5ab4ec61a05596cb33e469a61a870c Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Tue, 9 Jun 2015 23:34:50 -0700
Subject: [PATCH 4/8] XQuartz: Silence -Wunused-function

quartzKeyboard.c:741:1: warning: unused function 'macroman2ucs' [-Wunused-function,Unused Entity Issue]
macroman2ucs(unsigned char c)
^
1 warning generated.

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 3a6fa115759c787ec34483437021ad1e55c52423)
---
 hw/xquartz/quartzKeyboard.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hw/xquartz/quartzKeyboard.c b/hw/xquartz/quartzKeyboard.c
index 84e34d9..2fed593 100644
--- a/hw/xquartz/quartzKeyboard.c
+++ b/hw/xquartz/quartzKeyboard.c
@@ -737,6 +737,7 @@ LegalModifier(unsigned int key, DeviceIntPtr pDev)
     return 1;
 }
 
+#if !defined(__LP64__) || MAC_OS_X_VERSION_MIN_REQUIRED < 1050
 static inline UniChar
 macroman2ucs(unsigned char c)
 {
@@ -782,6 +783,7 @@ macroman2ucs(unsigned char c)
     if (c < 128) return c;
     else return table[c - 128];
 }
+#endif
 
 static KeySym
 make_dead_key(KeySym in)
-- 
2.6.1

